napari/utils/interactions.py

```diff
@@ -210,15 +210,37 @@ def hello_world(layer, event):
     'Enter': '↵',
 }
 
-
+joinchar = '-'
```

Comment:
Default valid Qt shortcut sequences use `+` as the joinchar, so is this something we want to change?